### PR TITLE
update for static provisioning yml

### PR DESCRIPTION
**Pull request by sandesh-desai**
update for static provisioning yml

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -403,9 +401,94 @@ pvc/pvc1     Bound     pv100     20Gi       RWO                          11m
 
 NAME                          READY     STATUS    RESTARTS   AGE       IP             NODE
 po/pod1                       1/1       Running   0          11m       10.128.1.53    cld6b16
+```
+
+**Static provisioning** is a feature that is native to Kubernetes and that allows cluster admins to make existing storage devices available to a cluster. As a cluster admin, you must know the details of the storage device, its supported configurations, and mount options.
+
+To make existing storage available to a cluster user, you must manually create the storage device, a PV, and a PVC.
```
Comment (nilangekarss) [inline docs/usage.md]: This statement will not be completely true, as few versions need POD to be created to created volume on 3PAR. Can you confirm what are those versions in our matrix .

File: docs/usage.md [hunk 2]
```diff
@@ -403,9 +401,94 @@ pvc/pvc1     Bound     pv100     20Gi       RWO                          11m
...
+
+Below is an example yaml specification to create Persistent Volumes using the HPE 3PAR FlexVolume driver. 
+
+```
+Note: If you have OpenShift installed, kubectl create and oc create commands can be used interchangeably when creating PVs, PVCs, and PODs.
+```
+
+Persistent volume Example
+The following creates a Persistent volume "pv-first" that provisions a compressed volume with the help of HPE 3PAR Docker Volume Plugin.
```
Comment (nilangekarss) [inline docs/usage.md]: Here you are claiming compressed volume creation but yaml doesn't reflect it.

File: ansible_3par_docker_plugin/samples/pv_static_example.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  name: pv-first
+spec:
+    capacity:
+      storage: 10Gi
+    accessModes:
+    - ReadWriteOnce
+    flexVolume:
+      driver: hpe.com/hpe
+      options: 
+        size: "10"
```
Comment (nilangekarss) [inline ansible_3par_docker_plugin/samples/pv_static_example.yml]: Do we need a size parameter here, does it impact?

Comment (sandesh-desai) [reply ansible_3par_docker_plugin/samples/pv_static_example.yml]: Yes,We need a size parameter in PV.

File: docs/usage.md (hunk 1 above)
Comment (sandesh-desai) [reply docs/usage.md]: I have make a generic statement to create static provisioning volume to PV,PVC and POD

File: docs/usage.md (hunk 2 above)
Comment (sandesh-desai) [reply docs/usage.md]: Updated the correct statement.

**Comment by sandesh-desai** [comment]
@nilangekarss : Updated the  comment section.

**Pull request merged**